cypress/e2e/attribute_validations/reporting.cy.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from failing the test
+  return false
+})
+
+describe('Add multiple clusters and search', () => {
+  // This will hold the fixture data and the flag for skipping the tests
+  let skipTest = false
+
+  before(function () {
+    // Load the fixture data before running any tests
+    cy.fixture('data').then((data) => {
+      this.data = data
+      // If mode is "matter", set skipTest flag to true
+      if (this.data.mode === 'matter') {
+        skipTest = true
+      }
+    })
+  })
+
+  it('create two endpoints and validate basic information', function () {
+    // Skip the test if the flag is true (mode is "matter")
+    if (skipTest) {
+      cy.log('Skipping test because mode is "matter"')
+      return // Skip the test if mode is 'matter'
+    }
+
+    // Continue with the test if the mode is not "matter"
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+    cy.setZclProperties()
+
+    // Create and validate the first endpoint
+    cy.fixture('data').then((data) => {
+      cy.addEndpoint(data.endpoint4, data.cluster1)
+    })
+    cy.get('.flex  > strong').should('contain', '#1')
+
+    // Select the "Power Configuration" cluster for the second endpoint
```

Comment:
is another `// Create and validate the second endpoint needed here?`
Just like the addition of the first endpoint.

Reply:
Yeah it's just the method of adding endpoints. It should be cleaned up and I'll make a follow up PR to fix cypress as a whole